### PR TITLE
WebUI: Make Test Cases look clickable.

### LINKED ISSUE #4003: Test Cases dont look like they are clickable
It's hard to make out if the test cases are cilckable and should point to wiki.
https://bodhi.fedoraproject.org/updates/FEDORA-2020-c014bea351

User should be able to clearly understand that the test cases are clickable, the colour of text for link to be changed maybe

### PR DESCRIPTION
**Pull request by richardgreg**
Fixes: #4003

Signed-off-by: Richard O. Gregory <richardgrecoson@gmail.com>

### CODE REVIEW

**Comment by richardgreg** [comment]
The original author intentionally made it 'not blue'. I wonder why though?

**Review by mattiaverga — CHANGES_REQUESTED** [review]
> The original author intentionally made it 'not blue'. I wonder why though?

I suppose that's because light blue text + small font size + white background is not the best for readability... I would suggest to change the `notblue` class with `font-weight-bolder` to enhance the readability.

Also, let's use f-strings while editing the text.

File: news/4003.dev [hunk 1]
```diff
@@ -0,0 +1 @@
+Make Test Cases look clickable.
```
Comment (mattiaverga) [inline news/4003.dev]: Ah, sorry, I didn't notice in my previous review the news fragment type: this isn't a development related change, a filename suffix of `.bug` should be fine.

**Review by mattiaverga — CHANGES_REQUESTED** [review]

**Review by cverna — APPROVED** [review]
LGTM

**Review by mattiaverga — APPROVED** [review]

**Pull request merged**